packages/python-requests/buildinfo.json

```diff
@@ -3,17 +3,17 @@
   "sources": {
     "requests": {
       "kind": "url",
-      "url": "https://pypi.python.org/packages/49/df/50aa1999ab9bde74656c2919d9c0c085fd2b3775fd3eca826012bef76d8c/requests-2.18.4-py2.py3-none-any.whl",
-      "sha1": "52ccdd6ee808bddd0c6eabc6eda79e79381266df"
+      "url": "https://files.pythonhosted.org/packages/ff/17/5cbb026005115301a8fb2f9b0e3e8d32313142fe8b617070e7baad20554f/requests-2.20.1-py2.py3-none-any.whl",
+      "sha1": "a38551c5a3baa3bfdaa89edb5f0f3e15b298ee6a"
     },
     "urllib3": {
       "kind": "url",
-      "url": "https://pypi.python.org/packages/63/cb/6965947c13a94236f6d4b8223e21beb4d576dc72e8130bd7880f600839b8/urllib3-1.22-py2.py3-none-any.whl",
-      "sha1": "ae6715ae61c34b72d5e0c3241abfb20c2c4d1313"
+      "url": "https://files.pythonhosted.org/packages/62/00/ee1d7de624db8ba7090d1226aebefab96a2c71cd5cfa7629d6ad3f61b79e/urllib3-1.24.1-py2.py3-none-any.whl",
```

Comment:
urllib3 is pinned to 1.22 in `setup.py`. Should that be updated too?

Reply:
I think so. I will update it.